### ISSUE TITLE
Decimal.isZero is not a function

### ISSUE THREAD

**Issue by pbcornell** [amzn/ion-js#223]
The following code throws:  `TypeError: this._value.isZero is not a function`:

```javascript
let dec = new ion.Decimal(0, 5);
let writer = ion.makeBinaryWriter();
writer.writeDecimal(dec);
writer.close();
```

**Comment by wesboyt** [comment]
https://github.com/amzn/ion-js/blob/master/src/IonDecimal.ts#L48
decimal only accepts LongInts as parameters right now, we can definitely add support for number as well.

**Comment by pbcornell** [comment]
This is believed to be resolved by #236
